tests/publiccloud/storage_perf.pm

```diff
@@ -24,28 +24,29 @@ use constant IODEPTH => 4;
 sub run {
     my ($self) = @_;
     my $reg_code = get_var('SCC_REGCODE');
-    my $runtime = get_var('PUBLIC_CLOUD_FIO_RUNTIME',  300);
-    my $size    = get_var('PUBLIC_CLOUD_FIO_SSD_SIZE', 100);
-    my $url     = get_var('PUBLIC_CLOUD_PERF_DB_URI');
+    my $runtime   = get_var('PUBLIC_CLOUD_FIO_RUNTIME', 300);
+    my $disk_size = get_var('PUBLIC_CLOUD_FIO_SSD_SIZE');
+    my $disk_type = get_var('PUBLIC_CLOUD_SSD_TYPE');
```

Comment:
```suggestion
    my $disk_type = get_var('PUBLIC_CLOUD_FIO_SSD_TYPE');
```

maybe your idea was that SSD type is actually not hardlinked to performance testing , but SSD size not hard linked in the same way :) so or remove FIO from size variable or add it here :)

Reply:
I'll add FIO, then it's more clear what it's used for.